test/provisioning/account_spec.js

```diff
@@ -93,6 +94,22 @@ describe('account API - Provisioning', function () {
     });
   });
 
+  it('Get a specific subAccount', async function () {
```

Comment:
@patrick-tolosa , minor, but consistency is important. 
In the previous test, it's called 'sub account' in this one 'subAccount' and the in the next 'sub-account'

Reply:
Looks like a mistake in the original, I'll open a PR to address it